### ISSUE TITLE
How do we best drive community engagement?

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#383]
In discussion about wider issue on #306, we should work out what we're trying to achieve and how with the online community.

**Comment by amcewen** [comment]
Some alternatives to Google Groups, that I've seen recommended but haven't used personally - https://emaildodo.com/ and https://www.mail-list.com/ (emaildodo is much better price-wise, but I've not looked beyond that)

**Comment by aubergine10** [comment]
At the very least, we should have a `Forum` or `Community` link on the top-level navigation on the website.

**Comment by aubergine10** [comment]
Two problems I see for new users on Discourse:

1. Can't sign up or login with GitHub - shows me an error page. Probably issue on Discourse end of things.
2. I can't post new topics. I imagine this is part of anti-spam mechanism, waiting for a user to gain rep first, but for anyone joining to post a question and not seeing the "new topic" button (or whatever it will be called) is an immediate turn off. At the risk of some spam, I'd enable new users to post at least 1 topic immediately after joining.

**Comment by aubergine10** [comment]
For reference - I can't add posts to Discourse:

![region capture 1](https://cloud.githubusercontent.com/assets/1386719/25880841/aca65558-3531-11e7-88fd-d0ae4f1d2f1d.png)

**Comment by DefProc** [comment]
> For reference - I can't add posts to Discourse:

Trust levels are a [feature, not a bug](https://meta.discourse.org/t/what-do-user-trust-levels-do/4924/3). Although it looks like I've changed the default setting. You should be able to now. 

We do also have the option to email discourse (in testing) which means no login is required.

**Comment by aubergine10** [comment]
Nope, still can't create new posts - only comments on existing posts.

If someone joins discourse, the specific reason they did so was to ask a question or make a suggestion. If they can't do that, it's a major blocker to community engagement. Imagine joining google group to ask a question and there being no "New topic" button. You'd just give up and leave.

The trust levels are a feature, but they should at least let one post be made - otherwise they are really just blocking engagement with the platform.

How it currently works:

* I feel like I'm banned from engaging from the get go
* I can post replies to existing topics, but can't start new ones
* I have literally no idea how to get my trust level up to the point where I can create topics
* How it feels: "Welcome do Discourse, we don't trust you, so you should probably just leave"

How I think it should work:

* Allow me to create at least one new topic:
    * Make my sign-up worthwhile
    * Let me know I can participate - that's why I signed up in the first place!
    * _New blood invigorates stale ponds_
* Throttle me severely until my trust level increases
    * Limit the rate at which I can create new topics and replies until I'm more trusted
    * When I hit the limit, make sure I know about it - show me a message on screen explaining why (eg. "New members can only create one post a day, we do this to discourage spammers")
* How it feels: "Welcome to Discourse, tread lightly at first while you find your way around."

If this results in a spam problem, the community needs to be able to deal with spam (much like you can on facebook - where if a few people report a post it gets taken down and then some admin/moderator can review those every couple of weeks or so).

**Comment by aubergine10** [comment]
GitHub SSO: confirmed as working! 👍

**Comment by aubergine10** [comment]
I now get a New Topic button over on the right  👍 

Any chance we can change its `background-colour` - make it green (like github comment button: `#28a745`) or something? It's practically invisible when just glancing at the screen.

**Comment by amcewen** [comment]
Any chance we can have the Discourse conversation over on its issue #306?  Please?

**Comment by aubergine10** [comment]
Yes, will do. But the comments above are still valid for consideration in context of the _"how"_ part of OP, regardless of _which system_ we eventually end up with or _what we're trying to achieve_: Community needs to know a) _where to go_ to engage and b) be _able to actually engage_ when they get there.
